CompuCell3D/core/Demos/bacterium_macrophage_new_style/Simulation/bacterium_macrophage_2D_steppables.py

```diff
@@ -61,6 +57,3 @@ def step(self, mcs):
                     pass
 
         self.plot_win.show_all_plots()
-
```

Comment:
I would leave it here. I added it on purpose here to illustrate how to stop simulation. What do you think @tjsego ?